src/app/modules/applications/components/servable-names/servable-names.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { ApplicationsFacade } from '@app/core/facades/applications.facade';
+import { ServablesFacade } from '@app/core/facades/servables.facade';
+import {
+  Application,
+  Servable,
+  ModelVersion,
+} from '@app/core/data/types';
+
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'hs-servable-names',
+  templateUrl: './servable-names.component.html',
+  styleUrls: ['./servable-names.component.scss']
+})
+export class ServableNamesComponent implements OnInit {
+  application$: Observable<Application>;
+  servables$: Observable<Servable[]>;
+
+  constructor(
+    private readonly applicationFacade: ApplicationsFacade,
+    private readonly servableFacade: ServablesFacade
+  ) { }
+
+  ngOnInit() {
+    this.application$ = this.applicationFacade.selectedApplication();
+    this.servables$ = this.servableFacade.allServables();
+  }
+
+  get servableNames(): string[] {
+    let modelVersion: ModelVersion;
+    let names: string[];
+    this.application$.subscribe(application => {
```

Comment:
Now on every changeDetection cycle angular will call your getter and subscribe to streams

- You can check it, add .pipe(tap(foo => console.log(foo))) to any stream
- If you want to chain stream you should you special functions as switchMap (depends on context and what you want) https://rxjs-dev.firebaseapp.com/api/operators/switchMap
- Also you can subscribe to steam in template {{ stream$ | async }}
- When you manually use subscribe you should add logic for unsubscribing (for example when component was destroyed)